# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/SharedSurfaceEGL.h
# Commit: edfb73b96d5a
# Full Hash: edfb73b96d5ad946bbdd9da2e98d07b1f4565e1f
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2019-06-15 21:44:24
# Regressor Bug: 1559524
# File Overlap Count: 2
# Description:
#   Bug 1559524 - Reduce calls to GLLibraryEGL::Get(). r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35103
# ==============================================================================

diff -r 0378898329b2 -r edfb73b96d5a gfx/gl/SharedSurfaceEGL.h
--- a/gfx/gl/SharedSurfaceEGL.h	Sat Jun 15 05:14:53 2019 +0000
+++ b/gfx/gl/SharedSurfaceEGL.h	Sat Jun 15 05:21:12 2019 +0000
@@ -39,7 +39,6 @@
 
  protected:
   mutable Mutex mMutex;
-  GLLibraryEGL* const mEGL;
   const GLFormats mFormats;
   GLuint mProdTex;
 
@@ -49,12 +48,10 @@
  protected:
   EGLSync mSync;
 
-  SharedSurface_EGLImage(GLContext* gl, GLLibraryEGL* egl,
-                         const gfx::IntSize& size, bool hasAlpha,
+  SharedSurface_EGLImage(GLContext* gl, const gfx::IntSize& size, bool hasAlpha,
                          const GLFormats& formats, GLuint prodTex,
                          EGLImage image);
 
-  EGLDisplay Display() const;
   void UpdateProdTexture(const MutexAutoLock& curAutoLock);
 
  public: